docs/tutorials/install-barbican.md

```diff
@@ -1,51 +1,13 @@
 ---
 sidebar_position: 7
 ---
-# How to Install OpenStack Barbican on your OpenMetal Cloud
+# How to Utilize OpenStack Barbican on your OpenMetal Cloud
 
-## Prerequisites
+## Prerequisite
 
-- SSH Access to your OMI Nodes
-- A [python virtual environment for kolla-ansible](https://openmetal.io/docs/manuals/operators-manual/day-4/kolla-ansible/prepare-kolla-ansible)
+- Familiarity with OpenStackClient
```

Comment:
Add a backlink on how to set up OpenStack client.
Pretty sure they need to install python-barbicanclient , no?